### PR TITLE
Add parser for AddRelationshipCommand

### PR DESCRIPTION
**Pull request by Tsenrae**
Added functionality for the parser to parse relationships

### CODE REVIEW

**Review by Bandov — COMMENTED** [review]
needs to add more tests for the relationship classes but the rest LGTM

**Review by Emberlynn-Loo — APPROVED** [review]
Code LGTM

File: src/main/java/seedu/address/model/person/relationship/RelationshipManager.java [hunk 1]
```diff
@@ -20,6 +20,10 @@ public void addRelationship(String relationshipType, Relationship relationship)
         relationships.computeIfAbsent(relationshipType, k -> new ArrayList<>()).add(relationship);
     }
 
+    public void deleteRelationship(String relationshipType, Relationship relationship) {
+        relationships.get(relationshipType).remove(relationship);
+    }
```
Comment (Emberlynn-Loo) [inline src/main/java/seedu/address/model/person/relationship/RelationshipManager.java]: You might want to put this in another pull request for DeleteRelatioshipCommand

File: src/main/java/seedu/address/logic/relationship/AddRelationshipCommand.java [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    /**
+     * Handles adding a relationship without roles based on the given command parts.
+     *
+     * @param commandParts The parts of the command to parse and execute
+     * @throws IllegalArgumentException If the person cannot be found or the relationship type is invalid.
+     */
+    private void handleAddRolelessRelationship(String[] commandParts) {
+        if (commandParts.length < 4) {
+            throw new IllegalArgumentException("Invalid command format");
+        }
```
Comment (Emberlynn-Loo) [inline src/main/java/seedu/address/logic/relationship/AddRelationshipCommand.java]: I think you can delete this as you have already checked it in the previous method

**Review by Emberlynn-Loo — COMMENTED** [review]
Overall LGTM

File: src/main/java/seedu/address/logic/relationship/AddRelationshipCommand.java
Comment (Emberlynn-Loo) [inline src/main/java/seedu/address/logic/relationship/AddRelationshipCommand.java:1]: May I know why it is not under the command package? Like just asking for a reason

File: src/main/java/seedu/address/logic/relationship/AddRelationshipCommandParser.java
Comment (Emberlynn-Loo) [inline src/main/java/seedu/address/logic/relationship/AddRelationshipCommandParser.java:1]: This class as well

**Review by Bandov — COMMENTED** [review]
LGTM
